### PR TITLE
Added Trailing_Zeroes.py

### PR DESCRIPTION
**Pull request by NEERAJAP2001**
solves #111

### CODE REVIEW

**Comment by NEERAJAP2001** [comment]
@rcoder23 Can This PR get a review?

File: Numerical/Trailing_Zeroes.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Input: number = 3
+# Output: 0
+# Explanation: 3! = 6, no trailing zero.
+
+# Input: number = 5
+# Output: 1
+# Explanation: 5! = 120
```
Comment (rohanbabbar04) [inline Numerical/Trailing_Zeroes.py:7]: Add the sample output after the code

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]

File: Numerical/Trailing_Zeroes.py (hunk 1 above)
Comment (NEERAJAP2001) [reply Numerical/Trailing_Zeroes.py:7]: I'm sorry I didn't understand that!
I'm assuming you want me to move the selected comments below the code, is it correct?

Comment (rohanbabbar04) [reply Numerical/Trailing_Zeroes.py:7]: YES

Comment (NEERAJAP2001) [reply Numerical/Trailing_Zeroes.py:7]: Hey, Actually I deleted the fork yesterday by mistake would you mind if I come with a new PR?

Comment (NEERAJAP2001) [reply Numerical/Trailing_Zeroes.py:7]: I've solved it Via #228 
Can you please review it now!

**Comment by NEERAJAP2001** [comment]
Solved via #228 
So, closing this PR

**Pull request closed without merging**